commons/src/main/scala/dagr/commons/reflect/ReflectionUtil.scala

```diff
@@ -21,16 +21,32 @@
  * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
  * THE SOFTWARE.
  */
-package dagr.core.util
+package dagr.commons.reflect
 
-import java.lang.reflect.Constructor
+import java.lang.reflect.{InvocationTargetException, Constructor}
+import java.nio.file.Path
 
-import dagr.DagrDef._
+import dagr.commons.util.StringUtil
+import dagr.commons.CommonsDef._
 
 import scala.annotation.ClassfileAnnotation
 import scala.reflect.ClassTag
 import scala.reflect.runtime.universe._
 import scala.reflect.runtime.{universe => ru}
+import scala.util.{Try, Success, Failure}
+
+object ReflectionUtilException {
+  def format(msg: String, e: Exception): String = {
```

Comment:
Since this is only used in one place that I can see, why not eliminate the object entirely and have the single line of concatenation be inside the secondary constructor for the exception?